common/build/eslint-plugin-fluid/package.json

```diff
@@ -12,7 +12,7 @@
 	"author": "Microsoft and contributors",
 	"main": "index.js",
 	"scripts": {
-		"build": "npm run build:readme",
+		"build": "tsc && npm run build:readme",
```

Comment:
I think if you want `tsc` to build, then you need a `tsconfig.json` file in the same directory as `package.json`. I see one under src/test, not sure if it supposed to be moved or if you need a new one

Reply:
I would also recommend making a separate PR first that updates this library to use TypeScript, if that is something we want to do (seems like a good idea to me). It's a pretty major change to the package, so it seems like a good candidate to be a separate PR. And it will make the real diffs in this PR easier to reason about.